apollo-idling-resource/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=apollo-idling-resource
```

Comment:
Hmm, yeah adding espresso makes it clear that this module is about supporting espresso testing. `apollo-idling-resource` sounds vague.